Rename a folder

MVrachev opened the pull request:
When we added TSLint spawn functionality -
https://github.com/vmware/precaution/pull/174 we made the mistake to add
test/fixtures/typescript folder which included javascript files. 
The name should be test/fixtures/javascript.

Signed-off-by: Martin Vrachev <mvrachev@vmware.com>

MVrachev commented:
> The folder should probably be named javascript, not js - for other languages we use full name of the language in lowercase

Yep, fixed.

ericwb commented:
Please update the commit message.  Don't refer to "I" and add some context on what you mean for historical purposes.  Like, what PR introduced the mistake you're referring to.

MVrachev commented:
> Please update the commit message. Don't refer to "I" and add some context on what you mean for historical purposes. Like, what PR introduced the mistake you're referring to.

Ohh... sorry. I forgot I shouldn't write messages only from my perspective.
I squashed the commits and fixed the message with more context.
